keyboards/sol/keymaps/brianweyer/keymap.c

```diff
@@ -0,0 +1,320 @@
+#include QMK_KEYBOARD_H
+#include "bootloader.h"
+#ifdef PROTOCOL_LUFA
+#include "lufa.h"
+#include "split_util.h"
+#endif
+#ifdef SSD1306OLED
+  #include "common/ssd1306.h"
+#endif
+
+extern keymap_config_t keymap_config;
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+extern uint8_t is_master;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum layer_number {
+    _QWERTY = 0,
+    _FN,
+    _ADJ
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  FN,
+  ADJ,
+  RGBRST
+};
+
+#define _______ KC_TRNS
+#define KC________ KC_TRNS
+#define KC_RGB_MOD RGB_MOD
+#define KC_FN FN
+#define KC_ADJ ADJ
+#define XXXXXXX KC_NO
```

Comment:
Line 42 is redundant and should be deleted.

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L31